pkg/printers/internalversion/printers.go

```diff
@@ -1328,11 +1330,14 @@ func printPersistentVolume(obj *api.PersistentVolume, options printers.PrintOpti
 	if obj.ObjectMeta.DeletionTimestamp != nil {
 		phase = "Terminating"
 	}
+	volumeMode := "<unset>"
+	if obj.Spec.VolumeMode != nil {
+		volumeMode = string(*obj.Spec.VolumeMode)
+	}
 
-	row.Cells = append(row.Cells, obj.Name, aSize, modesStr, reclaimPolicyStr,
```

Comment:
volumeMode must go in the same place as the header, which is currently the last column

Reply:
Done, seems the same reason with the next one